src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -7,6 +7,10 @@
     CinemaHall add(CinemaHall cinemaHall);
 
     CinemaHall get(Long id);
-    
+
     List<CinemaHall> getAll();
+
+    Boolean update(CinemaHall cinemaHall);
+
+    Boolean delete(Long id);
```

Comment:
Consider changing the return type of the `delete` method to `void` or throwing an exception if the deletion fails. Returning a Boolean might not be the best approach for error handling.